.jenkins/pytorch/test.sh

```diff
@@ -14,6 +14,17 @@ echo "Testing pytorch"
 if [ -n "${IN_CIRCLECI}" ]; then
   # TODO move this to docker
   pip_install unittest-xml-reporting coverage
+  # install lcov with 1.15 (gcc 9 requires lcov>1.14-2) outside of workplace, so we don't need to clean it up
```

Comment:
If we only need this for code coverage would it make sense to move this to the `"${BUILD_ENVIRONMENT}" == *coverage*` block down below?

We probably don't need to be installing this for every single test job we run.